docker-images/terraform/fry-helm-in-terraform

```diff
@@ -19,4 +19,4 @@ printenv | grep HELM_PLUGIN_ | while read -r i; do
   helm plugin install ${plugin_install_command}
```

Comment:
I'm feeling more reluctant to silence the installation of Helm plugins, but that should probably happen too. What do you think?

Reply:
I agree with you